test/sanity

```diff
@@ -32,3 +32,20 @@ else
     exit 1
 fi
 
+echo 'itIsOKForACamelCaseWordToContainNumerousCOMPONENTS' | aspell --camel-case -d en_US -a > tmp/res
+if cat tmp/res | fgrep '-'; then
```

Comment:
Sanity is meant for a few quick tests to make sure Aspell functions at all.  Specific unit tests should be in a separate file.  Please move these into the file `test/camel_case` and add a target for it in the Makefile.